### ISSUE TITLE
Fetch API cannot load shopify:admin/api/graphql.json. URL scheme "shopify" is not supported.

### ISSUE THREAD

**Issue by luklapp** [Shopify/shopify-app-bridge#280]
## Describe the bug

If I try to use direct API access in an embedded app, I receive this error:
```
Fetch API cannot load shopify:admin/api/graphql.json. URL scheme "shopify" is not supported.
```

### To Reproduce

Steps to reproduce the behaviour:

1. Follow these steps: https://shopify.dev/docs/api/app-bridge-library#direct-api-access

### Expected behaviour

Data from GraphQL API is returned

### Packages and versions
```
"@shopify/app": "^3.55.3",
"@shopify/app-bridge": "^3.7.10",
"@shopify/app-bridge-react": "^3.7.10",
```

Configuration in shopify.app.toml:
```
[...]

[access.admin]
direct_api_mode = "online"
embedded_app_direct_api_access = true

[...]
```

## Platform
Chrome on MacOS

**Comment by luklapp** [comment]
@charlesdobson Thank you, that was the issue! I skipped the getting started part, because I already have the app bridge inside of my app 🤦‍♂️ 

I think it would be nice to have this integrated also in `@shopify/app-bridge` and `@shopify/app-bridge-react`. The first thing I was looking for was an exported `fetch` inside of these packages. Then I tried to use `useAuthenticatedFetch`, which also didn't work. I think it would be intuitive for existing app bridge users to be able to use `useAuthenticatedFetch` for calls to `shopify:admin` :)
